.idea/codeStyles/Project.xml

```diff
@@ -54,4 +57,4 @@
       </arrangement>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Please keep the EOFs.

Reply:
This file is IDEA-generated. I'll change it now, but with the new code style change, it's going to be overwritten again.